tests/interactive/scroll-view-sizing.js

```diff
@@ -184,27 +177,28 @@ SizingIllustrator.prototype = {
 	    }
 	}
 
-	alloc.min_size = 0;
-	alloc.natural_size = 400;
+        return [0, 400];
     },
 
-    _getPreferredHeight: function (actor, forWidth, alloc) {
-        let children = this.actor.get_children();
+    vfunc_get_preferred_height(forWidth) {
+        let children = this.get_children();
 	for (let i = 0; i < children.length; i++) {
 	    let child = children[i];
 	    if (child == this.child) {
 		[this.minHeight, this.naturalHeight] = child.get_preferred_height(this.width);
 	    } else {
-		let [minWidth, naturalWidth] = child.get_preferred_width(-1);
-		child.get_preferred_height(naturalWidth);
+		let [minWidth, naturalWidth] = child.get_preferred_height(naturalWidth);
```

Comment:
The passed in `naturalWidth` seems undefined here. Also it does not make sense that the assigned variables are called `minWidth`/`naturalWidth` but we're querying the preferred height